git/operations.go

```diff
@@ -276,15 +282,43 @@ func changed(ctx context.Context, workingDir, ref string, subPaths []string) ([]
 	return splitList(out.String()), nil
 }
 
-func execGitCmd(ctx context.Context, args []string, config gitCmdConfig) error {
-	if trace {
-		print("TRACE: git")
-		for _, arg := range args {
-			print(` "`, arg, `"`)
+// traceGitCommand returns a log line that can be useful when debugging and developing git activity
+func traceGitCommand(args []string, config gitCmdConfig, stdout string, stderr string) string {
+	for _, exemptedCommand := range exemptedTraceCommands {
+		if exemptedCommand == args[0] {
+			return ""
 		}
-		println()
 	}
 
+	prepare := func(input string) string {
+		// remove any null characters
```

Comment:
These comments just (p)recapitulate what's obvious from the code, inaccurately. Better to let the code speak for itself.

Reply:
kk.